kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/net/dns/DnsServerConfigIP.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2011, 2016 Eurotech and/or its affiliates
+ * Copyright (c) 2011, 2016 Eurotech and/or its affiliates and others
```

Comment:
No, because this is a mixed copyright. Between Eurotech and Red Hat. And thus the copyright is "Eurotech (...) and others". But please check back with EMO if in doubt.